validation/oauth_config.xsd

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
How would we keep this consistent with https://gitlab.tableausoftware.com/connectivity/oauth-service/-/blob/master/src/main/resources/xsd/oauthConfig.xsd? Is it just copy the changes by hand currently?

Reply:
Yes, within the dev-<release> branch.  We also should audit on release boundaries like we do with capabilities today as well in case a change gets missed.

Reply:
Yes, unfortunately.